extensions/amp-story/1.0/amp-story-interactive.js

```diff
@@ -193,6 +200,21 @@ export class AmpStoryInteractive extends AMP.BaseElement {
     return this.optionElements_;
   }
 
+  /**
+   * Gets the interactive ID
+   * @private
+   * @return {string}
+   */
+  getInteractiveId_() {
+    if (!this.interactiveId_) {
+      const pageId = closest(dev().assertElement(this.element), (el) => {
+        return el.tagName.toLowerCase() === 'amp-story-page';
+      }).getAttribute('id');
+      this.interactiveId_ = `CANONICAL_URL+${pageId}`;
+    }
+    return this.interactiveId_;
+  }
```

Comment:
Took this out to a separate function since it's also used in the updateStoryStoreState_ function